R/learner_randomPlantedForest_regr_rpf.R

```diff
@@ -0,0 +1,93 @@
+#' @title Regression Random Planted Forest Learner
+#' @author jemus42
+#' @name mlr_learners_regr.rpf
+#'
+#' @description
+#' Random Planted Forest: A directly interpretable tree ensemble.
+#'
+#' Calls [randomPlantedForest::rpf()] from 'randomPlantedForest'.
+#'
+#' @inheritSection mlr_learners_classif.rpf Custom mlr3 defaults
```

Comment:
this also inherits the information about the L2 loss which is not applicable here.